internal/sourcecode/templates/nuxtjs/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Source: https://nuxtjs.org/deployments/koyeb#dockerize-your-application
+FROM node:lts as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN yarn install \
+  --prefer-offline \
+  --frozen-lockfile \
+  --non-interactive \
+  --production=false
+
+RUN yarn build
+
+RUN rm -rf node_modules && \
+  NODE_ENV=production yarn install \
+  --prefer-offline \
+  --pure-lockfile \
+  --non-interactive \
+  --production=true
+
+FROM node:lts
+
+WORKDIR /app
+
+COPY --from=builder /app  .
+
+ENV HOST 0.0.0.0
+ENV PORT 8080
+EXPOSE 80
```

Comment:
This line doesn't mean anything on Fly, so probably should be removed to avoid confusion.

Reply:
Thanks. Should be fine now

Reply:
> We should add a docs page to accompany this!

Going for it

Reply:
https://github.com/superfly/docs/pull/111